ansible/files/home/deployer/webhook/lib/deployer/payload.rb

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) 2010-2019  Sutou Kouhei <kou@clear-code.com>
+# Copyright (C) 2015  Kenji Okimoto <okimoto@clear-code.com>
+# Copyright (C) 2024  Takuya Kodama <otegami@clear-code.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+module Deployer
+  class Payload
+    RELEASE_WORKFLOWS= ["Package"].map(&:freeze).freeze
```

Comment:
```suggestion
    RELEASE_WORKFLOWS = ["Package"].map(&:freeze).freeze
```

そんなに`freeze`しなくてもいい気はするけど。。。
Rubyを使っているのにそんなに信じられない？というか。。。

Reply:
fix: [be34927](https://github.com/groonga/packages.groonga.org/pull/47/commits/be3492745a583e193dd0c415aa23a76dbe2adce6) そんなにfreezeしなくても良いと思うので、修正しました。